pallets/randomness/src/mock.rs

```diff
@@ -98,6 +98,13 @@ impl pallet_balances::Config for Test {
 	type WeightInfo = ();
 }
 
+pub struct RelayEpochIndex;
+impl GetEpochIndex<u64> for RelayEpochIndex {
+	fn get_epoch_index() -> (u64, Weight) {
+		(1u64, 0)
+	}
+}
+
 pub struct RelayRandomness;
 impl GetRelayRandomness<H256> for RelayRandomness {
 	fn get_current_block_randomness() -> (H256, Weight) {
```

Comment:
I am thinking whether the return value should be an option? I am wondering wether this is better than having a expect in the trait impl. I also think it might make sense for it to be an option, None potentially representing "block randomness not updated" case

Reply:
Not for us, obviously, but for other users of this pallet

Reply:
I like this better too